lib/smack/SmackRep.cpp

```diff
@@ -55,6 +59,36 @@ bool SmackRep::isInt(const llvm::Value* v) {
   return isInt(v->getType());
 }
 
+unsigned SmackRep::getIntSize(const llvm::Value* v) {
+  assert(isInt(v) && "wrong type, should be integer.");
```

Comment:
If this check is to be done, it should be done inside `getIntSize(const llvm::Type*)` instead, to cover both invocations.